k8s/setup_cluster.sh

```diff
@@ -24,7 +24,7 @@ gcloud beta container clusters create $CLUSTER_NAME \
     --network $NETWORK_NAME  --create-subnetwork=name=$NETWORK_NAME-subnet,range=10.0.0.0/24 \
     --services-ipv4-cidr=10.0.1.0/24 --default-max-pods-per-node=110 \
     --zone=$ZONE \
-    --cluster-version "1.16.12-gke.3" \
+    --cluster-version "1.15.12-gke.13" \
```

Comment:
I think 1.15 is already deprecated upstream, I would vote for using the latest possible version of gke

Reply:
> GKE continues to upgrade control planes in clusters on the Stable channel to 1.15.12-gke.2. Upgrades will proceed gradually over several GKE releases.
Based on [GKE Release Notes](https://cloud.google.com/kubernetes-engine/docs/release-notes), it's set as the new default kubernetes version. Also the version `1.16.12-gke.3` is not supported as mentioned here 
```
ERROR: (gcloud.beta.container.clusters.create) ResponseError: code=400, message=Node version "1.16.12-gke.3" is unsupported.
```